packages/apps/human-app/frontend/src/api/services/common/use-access-token-refresh.ts

```diff
@@ -32,19 +30,11 @@ export function useAccessTokenRefresh() {
       throwExpirationModalOnSignOut?: boolean;
     }) => {
       try {
-        const refetchAccessTokenSuccess = await apiClient(
-          apiPaths.worker.obtainAccessToken.path,
-          {
-            successSchema: signInSuccessResponseSchema,
-            options: {
-              method: 'POST',
-              body: JSON.stringify({
-                // eslint-disable-next-line camelcase -- camel case defined by api
-                refresh_token: browserAuthProvider.getRefreshToken(),
-              }),
-            },
-          }
-        );
+        const refetchAccessTokenSuccess = await apiClient.refreshToken();
+
+        if (!refetchAccessTokenSuccess) {
```

Comment:
Don't forget to update code here according to new implementation of `refreshToken`